### PR TITLE
preserve security info

### PR DESCRIPTION
**Pull request by fearthecowboy**

### CODE REVIEW

File: core/test/configuration.ts [hunk 1]
```diff
@@ -47,9 +47,6 @@ csharp:
     // output folder should be 'foo'
     assert.equal(cfg['output-folder'], 'foo');
 
-    // my-value should not get resolved here because the dependent variable is after
-    assert.equal(cfg['my-value'], '$(sample-value)');
```
Comment (daviwil) [inline core/test/configuration.ts:51]: Seems unrelated, why does it need to be removed?

Comment (fearthecowboy) [reply core/test/configuration.ts:51]: It was breaking a test (configuration doesn't work like that anymore)... I dunno why I didn't catch it before.
